File: algo/src/test/java/com/hartwig/actin/algo/evaluation/tumor/HasIntratumoralHemorrhageByMRITest.java
package com.hartwig.actin.algo.evaluation.tumor;

import static com.hartwig.actin.algo.evaluation.EvaluationAssert.assertEvaluation;

import com.hartwig.actin.TestDataFactory;
import com.hartwig.actin.algo.datamodel.EvaluationResult;

import org.junit.Test;

public class HasIntratumoralHemorrhageByMRITest {

    @Test
    public void canEvaluate() {
        HasIntratumoralHemorrhageByMRI function = new HasIntratumoralHemorrhageByMRI();

        assertEvaluation(EvaluationResult.UNDETERMINED, function.evaluate(TestDataFactory.createMinimalTestPatientRecord()));
    }
}

File: algo/src/test/kotlin/com/hartwig/actin/algo/evaluation/tumor/HasIntratumoralHemorrhageByMRITest.kt
package com.hartwig.actin.algo.evaluation.tumor

import com.hartwig.actin.TestDataFactory
import com.hartwig.actin.algo.datamodel.EvaluationResult
import com.hartwig.actin.algo.evaluation.EvaluationAssert.assertEvaluation
import org.junit.Test

class HasIntratumoralHemorrhageByMRITest {
    @Test
    fun canEvaluate() {
        val function = HasIntratumoralHemorrhageByMRI()
        assertEvaluation(EvaluationResult.UNDETERMINED, function.evaluate(TestDataFactory.createMinimalTestPatientRecord()))
    }
}